web/src/features/charts/elements/ValueAxis.tsx

```diff
@@ -12,28 +12,27 @@ function ValueAxis({ scale, width, formatTick }: ValueAxisProps) {
   const [y1, y2] = scale.range();
   return (
     <g
-      transform={`translate(${width - 1} -1)`}
+      transform={`translate(${width} 0)`}
       fill="none"
       fontSize="10"
       fontFamily="sans-serif"
       textAnchor="start"
       className="opacity-50"
-      strokeWidth={0.5}
```

Comment:
y-axis stroke width should only apply to the ticks, not the axis line itself (in order to keep consistent with x-axis)